compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/model/BIRNonTerminator.java

```diff
@@ -1429,4 +1431,34 @@ public void setRhsOperands(BIROperand[] operands) {
             this.nonGreedyChar = operands[1];
         }
     }
+
+
+    public static class RecordDefaultFPLoad extends BIRNonTerminator {
```

Comment:
Shall we add java doc similar to above non-termintors giving a small description what this does.